core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorManagerService.js

```diff
@@ -47,7 +47,26 @@ oppia.factory('SimpleEditorManagerService', [
             x: 0
           }
         }
-      }
+      },
+      ItemSelectionInput: {
+        CUSTOMIZATION_ARGS:  {
+          choices: {
+            value: ['<p>Choice 1</p>']
+          },
+          minAllowableSelectionCount: {
+            value: 1
+          },
+          maxAllowableSelectionCount: {
+            value: 1
+          }
+        },
+        FIRST_ANSWER_GROUP_RULE: {
+          type: 'Equals',
+          value: {
+            x: 0
```

Comment:
Weird value for checkbox input. Shouldn't it be a list?

Reply:
Fixed!